pymeasure/instruments/anritsu/anritsuMS9710C.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, adapter, name="Anritsu MS9710C Optical Spectrum Analyzer", **
     #  Mappings #
     #############
     ONOFF = ["ON", "OFF"]
-    ONOFF_MAPPING = {True: 'ON', False: 'OFF', 1: 'ON', 0: 'OFF'}
+    ONOFF_MAPPING = {True: 'ON', False: 'OFF', 1: 'ON', 0: 'OFF', 'ON': 'ON', 'OFF': 'OFF'}
```

Comment:
Agreed, I also prefer a single-key interface (True/False) -- otherwise you just accumulated different key pairs ~endlessly, because why not _also_ allow `yes`/`no`, `on`/`off`,...

Reply:
Oh, and please revert this change.